setup.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# encoding: utf-8
+#
+# See COPYRIGHT and LICENSE files at the top of the source tree.
+#
+
+from setuptools import setup, find_packages
+import os
+
+
+PACKAGENAME = 'community_mailbot'
+DESCRIPTION = 'Friendly mail forwarding bot for community.lsst.org'
+LONG_DESCRIPTION = ''
+AUTHOR = 'Jonathan Sick'
+AUTHOR_EMAIL = 'jsick@lsst.org'
+LICENSE = 'MIT'
+URL = 'https://github.com/lsst-sqre/community_mailbot'
+
+
+def read(filename):
+    full_filename = os.path.join(
+        os.path.abspath(os.path.dirname(__file__)),
+        filename)
+    return open(full_filename).read()
```

Comment:
Should this `open` take an encoding? I know this is just a `setup.py` but ideally I'd like the file to be closed when we have done reading it (with a `with`).